src/engine/mqttControl.js

```diff
@@ -50,24 +51,26 @@ class MqttControl extends EventEmitter{
         this.runtime.on('SEQUENCE_STARTED', data => {
             this.playLightSequence(data);
         });
+        this.runtime.on('ADD_SUB_MQTTCONTROL', topic => {
+            this.addUserSub(topic);
+        });
     }
-    
 
-    static findSatelliteName (name) {
+    static findSatelliteName(name) {
         switch (name) {
-        case `${this.props.satOneName}`:
-            return 1;
-        case `${this.props.satTwoName}`:
-            return 2;
-        case `${this.props.satThreeName}`:
-            return 3;
-        case `${this.props.satFourName}`:
-            return 4;
+            case `${this.props.satOneName}`:
```

Comment:
This looks like it has a) tabs instead of spaces and b) incorrect indentation.  can you check that..

Reply:
fixed now